pkgs/development/python-modules/hdlparse/default.nix

```
@@ -0,0 +1,22 @@
+{stdenv,buildPythonPackage, fetchPypi}:
+
+buildPythonPackage rec {
+	pname = "hdlparse";
```

two spaces is standard indention
```suggestion
	 pname = "hdlparse";
```